### ISSUE TITLE
.ruby-version differs in VM vs App

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/sessionizer#14]
See:
https://github.com/curationexperts/sessionizer/blob/master/src/.ruby-version (2.2.2)
vs
https://github.com/curationexperts/sessionizer/blob/master/.ruby-version (2.2.3)

**Comment by jcoyne** [comment]
Yeah, I don't think the one at the root does anything, it probably should be removed.
